src/Simulation/Type3Core/Microsoft.Quantum.Type3.Core.csproj

```diff
@@ -0,0 +1,11 @@
+﻿<Project Sdk="Microsoft.Quantum.Sdk/0.14.2012122063-beta">
+
+  <Import Project="..\TargetDefinitions\TargetPackages\Type3.Package.props" />
+
+  <PropertyGroup>
+    <Description>Type2 Targeting support for the Q# programming language.</Description>
```

Comment:
Did you mean Type3 here?

Reply:
Whoops! I sure did. Thanks for catching!